web/src/regression/apiDriver.ts

```diff
@@ -0,0 +1,118 @@
+/**
+ * Provides convenience functions for interacting with the Sourcegraph API from tests.
+ */
+
+import { gql } from '../../../shared/src/graphql/graphql'
+import * as GQL from '../../../shared/src/graphql/schema'
+import { GraphQLClient } from './api'
+
+export class APIDriver {
+    constructor(public gqlClient: GraphQLClient) {}
+
+    /**
+     * Block until all repositories in the list exist
```

Comment:
This comment is a bit inaccurate in JS because nothing is really "blocking" unless it's CPU bound.
```suggestion
     * Wait until all repositories in the list exist
```